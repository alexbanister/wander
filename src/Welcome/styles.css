.welcome {
  display: flex;
  align-items: center;
  justify-content: center;
  flex-wrap: wrap;
  padding: 3em; }
  .welcome h3 {
    font-size: 3em;
    color: #F6D22F;
    margin: 0;
    padding: 0; }
  .welcome p {
    font-size: 1.5em; }
  .welcome .auth {
    display: flex;
    justify-content: center; }
  .welcome .form-block {
    background: #405F73;
    border: 3px solid #AEE173;
    padding: 1em;
    margin: 2em; }
