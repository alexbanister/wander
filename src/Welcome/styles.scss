@import "../variables.scss";

.welcome {
  display: flex;
  align-items: center;
  justify-content: center;
  flex-wrap: wrap;
  padding: 3em;
  h3 {
    font-size: 3em;
    color: $highlight;
    margin: 0;
    padding: 0;
  }
  p {
    font-size: 1.5em;
  }
  .auth {
    display: flex;
    justify-content: center;
  }
  .form-block {
    background: $body-light;
    border: 3px solid $highlight-dark;
    padding: 1em;
    margin: 2em;
  }
}
