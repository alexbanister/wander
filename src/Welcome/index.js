import React, { Component } from 'react';
import User from '../User/';
import './styles.css';

export default class Welcome extends Component {
  render() {
    return (
      <div className='welcome'>
        <h3>Make your Bucket List,</h3>
        <h3>your completed list</h3>
        <p>{
          'One place to save all the places you want to go and quickly search' +
          'for flights to all your destinations at one. You decide when you have' +
          "time to travel and how you want to travel and we'll show you all your" +
          'options and how well they match what you want.'
        }</p>
        <div className='auth'>
          <span className='form-block'>
            <h4>Login</h4>
            <User />
          </span>
          <span className='form-block'>
            <h4>Signup</h4>
            <User form='signup' />
          </span>
        </div>
      </div>
    );
  }
}
