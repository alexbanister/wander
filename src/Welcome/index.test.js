import { shallow } from 'enzyme';
import Welcome from './index';
import React from 'react';


describe('Welcome', () => {
  it('should always match the snapshot', () => {
    const wrapper = shallow(<Welcome />);

    expect(wrapper).toMatchSnapshot();
  });
});
