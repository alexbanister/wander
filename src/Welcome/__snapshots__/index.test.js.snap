// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome should always match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Welcome />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h3>
          Make your Bucket List,
</h3>,
        <h3>
          your completed list
</h3>,
        <p>
          One place to save all the places you want to go and quickly searchfor flights to all your destinations at one. You decide when you havetime to travel and how you want to travel and we'll show you all youroptions and how well they match what you want.
</p>,
        <div
          className="auth"
>
          <span
                    className="form-block"
          >
                    <h4>
                              Login
                    </h4>
                    <Connect(User) />
          </span>
          <span
                    className="form-block"
          >
                    <h4>
                              Signup
                    </h4>
                    <Connect(User)
                              form="signup"
                    />
          </span>
</div>,
      ],
      "className": "welcome",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "Make your Bucket List,",
        },
        "ref": null,
        "rendered": "Make your Bucket List,",
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "your completed list",
        },
        "ref": null,
        "rendered": "your completed list",
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "One place to save all the places you want to go and quickly searchfor flights to all your destinations at one. You decide when you havetime to travel and how you want to travel and we'll show you all youroptions and how well they match what you want.",
        },
        "ref": null,
        "rendered": "One place to save all the places you want to go and quickly searchfor flights to all your destinations at one. You decide when you havetime to travel and how you want to travel and we'll show you all youroptions and how well they match what you want.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <span
              className="form-block"
>
              <h4>
                            Login
              </h4>
              <Connect(User) />
</span>,
            <span
              className="form-block"
>
              <h4>
                            Signup
              </h4>
              <Connect(User)
                            form="signup"
              />
</span>,
          ],
          "className": "auth",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h4>
                  Login
</h4>,
                <Connect(User) />,
              ],
              "className": "form-block",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "Login",
                },
                "ref": null,
                "rendered": "Login",
                "type": "h4",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "class",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "span",
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h4>
                  Signup
</h4>,
                <Connect(User)
                  form="signup"
/>,
              ],
              "className": "form-block",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "Signup",
                },
                "ref": null,
                "rendered": "Signup",
                "type": "h4",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "class",
                "props": Object {
                  "form": "signup",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "span",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            Make your Bucket List,
</h3>,
          <h3>
            your completed list
</h3>,
          <p>
            One place to save all the places you want to go and quickly searchfor flights to all your destinations at one. You decide when you havetime to travel and how you want to travel and we'll show you all youroptions and how well they match what you want.
</p>,
          <div
            className="auth"
>
            <span
                        className="form-block"
            >
                        <h4>
                                    Login
                        </h4>
                        <Connect(User) />
            </span>
            <span
                        className="form-block"
            >
                        <h4>
                                    Signup
                        </h4>
                        <Connect(User)
                                    form="signup"
                        />
            </span>
</div>,
        ],
        "className": "welcome",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "Make your Bucket List,",
          },
          "ref": null,
          "rendered": "Make your Bucket List,",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "your completed list",
          },
          "ref": null,
          "rendered": "your completed list",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "One place to save all the places you want to go and quickly searchfor flights to all your destinations at one. You decide when you havetime to travel and how you want to travel and we'll show you all youroptions and how well they match what you want.",
          },
          "ref": null,
          "rendered": "One place to save all the places you want to go and quickly searchfor flights to all your destinations at one. You decide when you havetime to travel and how you want to travel and we'll show you all youroptions and how well they match what you want.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <span
                className="form-block"
>
                <h4>
                                Login
                </h4>
                <Connect(User) />
</span>,
              <span
                className="form-block"
>
                <h4>
                                Signup
                </h4>
                <Connect(User)
                                form="signup"
                />
</span>,
            ],
            "className": "auth",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h4>
                    Login
</h4>,
                  <Connect(User) />,
                ],
                "className": "form-block",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Login",
                  },
                  "ref": null,
                  "rendered": "Login",
                  "type": "h4",
                },
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "class",
                  "props": Object {},
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": "span",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h4>
                    Signup
</h4>,
                  <Connect(User)
                    form="signup"
/>,
                ],
                "className": "form-block",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Signup",
                  },
                  "ref": null,
                  "rendered": "Signup",
                  "type": "h4",
                },
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "class",
                  "props": Object {
                    "form": "signup",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": "span",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
